workers/unity/Packages/com.improbable.gdk.querybasedinteresthelper/Constraint.cs

```diff
@@ -0,0 +1,277 @@
+using System.Collections.Generic;
+using System.Linq;
+using Improbable.Gdk.Core;
+using UnityEngine;
+using UnityEngine.Assertions;
+
+namespace Improbable.Gdk.QueryBasedInterest
+{
+    /// <summary>
+    ///     Utility class to help construct QueryConstraint objects.
+    /// </summary>
+    public static class Constraint
+    {
+        private static readonly List<ComponentInterest.QueryConstraint> EmptyList
+            = new List<ComponentInterest.QueryConstraint>();
+
+        private static ComponentInterest.QueryConstraint Default()
+        {
+            return new ComponentInterest.QueryConstraint
+            {
+                AndConstraint = EmptyList,
+                OrConstraint = EmptyList
+            };
+        }
+
+        /// <summary>
+        ///     Creates a QueryConstraint object with a Sphere constraint.
+        /// </summary>
+        /// <param name="radius">
+        ///     Radius of the Sphere constraint.
+        /// </param>
+        /// <param name="center">
+        ///     Center of the Sphere constraint.
+        /// </param>
+        /// <returns>
+        ///     A QueryConstraint object.
+        /// </returns>
+        public static ComponentInterest.QueryConstraint Sphere(double radius, Vector3 center)
```

Comment:
Is it worth providing overrides with explicit `x`, `y`, `z` args?

Reply:
yes - that's a good idea